### ISSUE TITLE
Documentation improvement - Help wanted!

### ISSUE THREAD

**Issue by pvdlg** [semantic-release/semantic-release#653]
Most of the documentation has been rewritten recently. The content minimum content we need is there but there is a lot that can be done to improve it.

If you'd like to contribute to the semantic-release project, helping with the following would be much appreciated:
- Fix typo, misspelling and gramatical mistakes
- Improve language: make short and simpler sentences, improve clarity
- Harmonize the style and format, especially regarding `code quote`, markdown table, capitalization of brands, links

See [CONTRIBUTING.md#documentation](https://github.com/semantic-release/semantic-release/blob/caribou/CONTRIBUTING.md#documentation) for all the details.

All the repository in the [semantic-release organization](https://github.com/semantic-release) need some improvement regarding the documentation!

Thanks!

**Comment by ts-npm** [comment]
I could help with this.

**Comment by gr2m** [comment]
Pull requests welcome :) Thanks for your help!

**Comment by whosford** [comment]
I have a question about the dryRun configuration documentation. It states the following for the default value.

`Default: false if running in a CI environment, false otherwise`

This makes it sound as though the default is always false, is this correct?

**Comment by gr2m** [comment]
yes, I think that should be 

> Default: `false` if running in a CI environment, `true` otherwise

But maybe double check the code to make sure :)

**Comment by EfstathiadisDimitris** [comment]
Do you guys still need help. I can jump on this in the weekend..

**Comment by pvdlg** [comment]
@EfstathiadisDimitris yes, I think the documentation can always be improved. I wrote most of the doc and I'm not a native English speaker not a good documentation writer in general.
So help would be greatly appreciated. A lot of doc has been added recently on the `beta` branch, so you might want to look at that as it's probably what needs the most improvements.

**Comment by ray1981jacob** [comment]
Is anything else know if I need to do

**Comment by abdelrahmanahmed** [comment]
I going to write an article on how to use semantic-release, is this gonna help you with your documentation?

**Comment by JoA-MoS** [comment]
I saw commitizen was removed in #1255 the contributing guide still references this as well as an npm script that doesn't exist. What guidelines do you want to provide contributors? Should it just say use the Angular Commit conventions and link to that page?

**Comment by pvdlg** [comment]
The paragraph https://github.com/semantic-release/semantic-release/blob/master/CONTRIBUTING.md#commits should be just removed. The rest of the doc details the convention already.

**Comment by seeliang** [comment]
do we have a document about how to make `beta` branch generate `beta` build?

**Comment by pvdlg** [comment]
Yes:
- https://github.com/semantic-release/semantic-release/blob/beta/docs/usage/workflow-configuration.md#workflow-configuration
- https://github.com/semantic-release/semantic-release/tree/beta/docs/recipes#release-workflow

**Comment by seeliang** [comment]
@pvdlg , thanks, great doc, somehow it is not indexed by google, shame

**Comment by edmundo096** [comment]
Not sure but there may be an order error on an example given on https://github.com/semantic-release/semantic-release/blob/master/docs/usage/plugins.md#plugins-declaration-and-execution-order

The list currently has

- execute the `verifyConditions` implementation of @semantic-release/npm then @semantic-release/git
- execute the `analyzeCommits` implementation of @semantic-release/commit-analyzer
- **execute the `prepare` implementation of @semantic-release/npm then @semantic-release/git**
- **execute the `generateNotes` implementation of @semantic-release/release-notes-generator**
- execute the `publish` implementation of @semantic-release/npm

but as far I can understand according the steps defined on the top of the doc, `generateNotes` should go before `prepare`.

May not be a good place to comment it but better somewhere than missing it for later.

**Comment by akshayhat** [comment]
Do you guys still need help related to documentation improvement ? If yes , Please let me know if anything specific needs attention.

**Comment by Matheus-RR** [comment]
I'd like to help with this. I was reading through the docs and spotted a few things in `docs/usage/ci-configuration.md`:

- **Outdated CI services listed**: Wercker was discontinued after Oracle acquired it, and Codeship was shut down by CloudBees in 2022. Both are still listed as active options.
- **GoCD link** points to general concepts rather than pipeline configuration
- Missing some widely-used CI services like Bitbucket Pipelines and Azure DevOps

Happy to submit a PR to update the CI configuration page — remove defunct services, fix links, and add any missing providers. Would also review the rest of the usage docs for similar issues while I'm at it.
